spec/lib/patches/db/pg_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'pg'
+
+require './lib/patches/db/pg'
+
+require 'spec_helper'
+
+describe PG::Connection do
+  let(:conn) { described_class.open(dbname: 'test') }
```

Comment:
This is a custom local database, made just to test this. @SamSaffron can you think of a better way of doing this, or should I write some set-up code for this db in case it's not been created?